interface/L1TTau.hh

```diff
@@ -0,0 +1,32 @@
+#ifndef ICHiggsTauTau_L1TTau_hh
+#define ICHiggsTauTau_L1TTau_hh
+
+#include "UserCode/ICHiggsTauTau/interface/L1TObject.hh"
+
+#include <vector>
+#include "Math/Vector4D.h"
+#include "Math/Vector4Dfwd.h"
+#include "Rtypes.h"
+
+namespace ic {
+  
+  class L1TTau : public ic::L1TObject {
+  public:
+    L1TTau();
+    virtual ~L1TTau();
+    virtual void Print() const;
+    
+  #ifndef SKIP_CINT_DICT
+  public:
+    ClassDef(L1TTau, 1);
+  #endif
+    
+  public:
+    short int isolation;
+    
+  };
+  
+  typedef std::vector<ic::L1TTau> L1TTauCollection;
+}
+
+#endif
```

Comment:
So as already discussed with Riccardo when he was trying to solve a similar problem I'm not such a huge fan of adding this many objects. Others should comment but we might have to postpone a decision on this...